models/Attribution.js

```diff
@@ -116,7 +121,27 @@ class Attribution extends BaseModel {
     return await Attribution.query().findById(id)
   }
 
-  // TODO: delete by publicationIdb
+  static async getAttributionByPubId (publicationId /*: string */) /*: any */ {
+    if (publicationId == null) {
+      throw Error(`Your publicationId cannot be null`)
+    }
+
+    return Attribution.query().where('publicationId', publicationId)
```

Comment:
you need to have an 'await' there
`return await Attribution.query().where(....)`